src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
+    let resultsArray = [];
+    const loop = list.map( (item) => {
+      if(item.month === givenMonth && item.type === givenType) {
+        resultsArray.push(item);
+      };
+    });
+    if(resultsArray.length) {
```

Comment:
Tell a lie - I *have* used .some before, though I only remembered that once I looked up the official documentation. Must have been the heatwave talking!!